### PR TITLE
Docs di container

### PR DESCRIPTION
**Pull request by softark**
I'm not very sure whether I got the concept right or not. You have to review it.

### CODE REVIEW

File: docs/guide/concept-di-container.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependency injection:
 * Constructor injection;
 * Setter and property injection;
 * PHP callable injection.
+* Method signiture injection.
```
Comment (cebe) [inline docs/guide/concept-di-container.md]: `signiture`  ->  `signature`

**Comment by cebe** [comment]
why did you change the controller injection docs? related to #9785, which as far as I see is not implemented.

**Comment by softark** [comment]
Oh, sorry, I thought it has been implemented already.
Should I revert it to "Controller action injection" ?

**Comment by samdark** [comment]
Yes.

**Comment by softark** [comment]
OK, I reverted "Method signature injection" to "Controller action injection".

**Comment by samdark** [comment]
Thanks!

**Pull request merged**